Sources/RInAppMessaging/Views/FullView.swift

```diff
@@ -210,7 +207,7 @@ internal class FullView: UIView, FullViewType, RichContentBrowsable {
 
         switch mode {
         case .fullScreen:
-            contentView.topAnchor.constraint(equalTo: topAnchor).isActive = true
```

Comment:
Why it was not using `safeAreaLayoutGuide` before? 🤔 I can't remember. Maybe it was for image layout. Anyway let's keep this change

Reply:
Do you mean revert this change?

And before the image used to be positioned into the Safe Area but now we have a spacer with the ⓧ above the image:

<img align="left" src="https://user-images.githubusercontent.com/3298414/142163725-c760497c-9eb3-42c5-9e9a-465309335df3.png" width="240"><img align="right" src="https://user-images.githubusercontent.com/3298414/141994264-6c39e0b3-52e2-40e3-a3c0-86c692e41f01.png" width="240">

Reply:
That's what I thought. Your change is correct